lib/modules/idea-overview-widgets/index.js

```diff
@@ -262,7 +356,16 @@ module.exports = {
     const superLoad = self.load;
 		self.load = function(req, widgets, next) {
 			widgets.forEach((widget) => {
-        widget.ideas = req.data.ideas;
+        widget.ideas = req.data.ideas.map((idea) => {
+          if (widget.mapCustomStatuses && widget.customStatusMapping.includes(idea.status) && widget['map' + idea.status]) {

```

Comment:
Proposal: For better readability it's nicer to move this if statement to a separate function:
For example:
`self.hasCustomStatusMapping(widget, idea.status)`

Reply:
Nice proposal, I added this in. I also set the idea.status directly instead of mappedStatus, since the ideas is added to the widget and not the request.